NickvisionMoney.GNOME/Views/TransferDialog.cs

```diff
@@ -73,6 +73,15 @@ private TransferDialog(Gtk.Builder builder, TransferDialogController controller,
                 Validate();
             }
         };
+        
+        _conversionRateGroup.OnNotify += (sender, e) =>
+        {
+            if (e.Pspec.GetName() == "visible")
+            {
+                PopulateRateFields();
+            }
+        };
+        
```

Comment:
Here and in TransferDialogController:
Please don't use blank lines as separator inside functions. If you feel like you really want to visually separate some part in a function, use comments.